examples/galaxies/plot_luminosity.py

```diff
@@ -0,0 +1,76 @@
+"""
+Schechter Luminosity Function
+=============================
+
+The distribution of galaxy luminosities is widely observed to follow the
+Schechter luminosity function. SkyPy implements methods to sample from a
+general Schechter function which are demonstrated here.
+
+"""
+
+# %%
+# Red and Blue Galaxies
+# ---------------------
+# Galaxies are often catagorised into two broad populations based on their
+# photometry. The slope `alpha` of the Schechter luminosity function is
+# observed to be different for these "red" and "blue" galaxies. Here we sample
+# one million B-band luminosities each from two Schecter functions with
+# different slopes and compare their distributions. `x_min` and `x_max` give
+# the range of sampled luminosities in units of `scale`; in this case the
+# B-band luminosity of the sun. With an appropriate choice of scale, this
+# function could also be used to sample e.g. the stellar masses of galaxies or
+# the masses of dark matter halos.
+
+from skypy.galaxy.luminosity import luminosity_from_absolute_magnitude
+from skypy.utils.random import schechter
+
+alpha_red, alpha_blue = -0.5, -1.3
```

Comment:
This is OK.
Just a suggestion. We could make these values more realistic. According to [Peng et al. 2001](https://arxiv.org/abs/1003.4747),  the faint slope  for (mass quenched) red galaxies differs from the blue population by one.
```suggestion
alpha_red, alpha_blue = -0.5, -1.5
```